### PR TITLE
prepare-changelog-1.5.10-beta02

### PR DESCRIPTION
**Pull request by Walingar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+* Improve rendering performance
+  * [Avoid redundant compositing](https://github.com/JetBrains/compose-multiplatform-core/pull/813)
+  * [Don't send redundant synthetic moves](https://github.com/JetBrains/compose-multiplatform-core/pull/819)
+  * [Postpone `CAMetalDrawable` acquisition](https://github.com/JetBrains/compose-multiplatform-core/pull/820)
+  * [Move frame encoding to separate thread when possible](https://github.com/JetBrains/compose-multiplatform-core/pull/829)
+
+### Fixes
+* [Expanded `ModalBottomSheet`: scrim doesn't occupy complete screen](https://github.com/JetBrains/compose-multiplatform/issues/3701)
+* [Fix interop view intercepting touches for popups](https://github.com/JetBrains/compose-multiplatform-core/pull/835) 
+* [Fix applying WindowInsets inside Popup/Dialog](https://github.com/JetBrains/compose-multiplatform-core/pull/832)
```
Comment (MatkovIvan) [inline CHANGELOG.md:26]:
```suggestion
* [Fix applying `WindowInsets` inside `Popup`/`Dialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/832)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+
+### Features
+
+* Improve accessibility support
+  * [Implement `Role.DropdownList` via `AccessibleRole.COMBO_BOX`](https://github.com/JetBrains/compose-multiplatform-core/pull/822)
+  * [Fix Compose `Role.Tab` to correctly translate to Java's `AccessibleRole.PAGE_TAB`](https://github.com/JetBrains/compose-multiplatform-core/pull/821)
+  * [Implement support for `SemanticsProperties.ProgressBarRangeInfo`](https://github.com/JetBrains/compose-multiplatform-core/pull/830)
+
+### Fixes
+* [Can't type in TextField placed in `ModalBottomSheet`](https://github.com/JetBrains/compose-multiplatform/issues/3703)
```
Comment (MatkovIvan) [inline CHANGELOG.md:39]:
```suggestion
* [Can't type in `TextField` placed in `ModalBottomSheet`](https://github.com/JetBrains/compose-multiplatform/issues/3703)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+* [Implement `defaultTimePickerLayoutType` based on screen orientation](https://github.com/JetBrains/compose-multiplatform-core/pull/817)
+* [Add an option to disable insets in `Popup`/`Dialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/833)
+* [Commonize insets `Modifier`'s \(additionally to `WindowInsets.*`\)](https://github.com/JetBrains/compose-multiplatform/issues/3563)
+
+### Fixes
+* [`ExposedDropdownMenuBox.onExpandedChange` was not recomposed](https://github.com/JetBrains/compose-multiplatform/issues/3686)
+
+## iOS
+
+### Features
```
Comment (igordmn) [inline CHANGELOG.md:16]: Waiting for https://github.com/JetBrains/compose-multiplatform-core/pull/831

**Review by igordmn — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+  * [Avoid redundant compositing](https://github.com/JetBrains/compose-multiplatform-core/pull/813)
+  * [Don't send redundant synthetic moves](https://github.com/JetBrains/compose-multiplatform-core/pull/819)
+  * [Postpone `CAMetalDrawable` acquisition](https://github.com/JetBrains/compose-multiplatform-core/pull/820)
+  * [Move frame encoding to separate thread when possible](https://github.com/JetBrains/compose-multiplatform-core/pull/829)
+
+### Fixes
+* [Expanded `ModalBottomSheet`: scrim doesn't occupy complete screen](https://github.com/JetBrains/compose-multiplatform/issues/3701)
+* [Fix interop view intercepting touches for popups](https://github.com/JetBrains/compose-multiplatform-core/pull/835) 
+* [Fix applying `WindowInsets` inside `Popup`/`Dialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/832)
+* [Fix incorrect text selection by double tap in `TextField`](https://github.com/JetBrains/compose-multiplatform/issues/2682)
```
Comment (igordmn) [inline CHANGELOG.md:27]: Better to move it to Features as "Double tap and triple tap gesture handling in textfields on iOS"

**Pull request merged**